src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -303,6 +319,16 @@ void DlgPrefRecord::slotSliderQuality() {
     // Settings are only stored when doing an apply so that "cancel" can actually cancel.
 }
 
+// Set 'Enable File Annotation in CUE file' checkbox value depending on 'Create a CUE file' checkbox value
+void DlgPrefRecord::updateCueEnabled() {
+    if (CheckBoxRecordCueFile->isChecked()) {
+        CheckBoxUseCueFileAnnotation->setEnabled(true);
+    } else {
+        CheckBoxUseCueFileAnnotation->setEnabled(false);
+        CheckBoxUseCueFileAnnotation->setChecked(false);
+    }
```

Comment:
Do we have to uncheck the `File Annotation` checkbox? Disabling make sense tho, to highlight the dependency.

```suggestion
   CheckBoxUseCueFileAnnotation->setEnabled(CheckBoxRecordCueFile->isChecked());
```